components/Button/index.css

```diff
@@ -0,0 +1,127 @@
+button,
```

Comment:
Is there a reason you duplicated this file?

Reply:
Mainly because I didn't want to clobber / move your original stuff yet. But, this could totally be a new home for the button styles